solutions/src/main/java/ru/sberbank/school/task02/FxClientControllerImpl.java

```diff
@@ -0,0 +1,97 @@
+package ru.sberbank.school.task02;
+
+import lombok.NonNull;
+import ru.sberbank.school.task02.exception.WrongSymbolException;
+import ru.sberbank.school.task02.util.ClientOperation;
+import ru.sberbank.school.task02.util.Symbol;
+import ru.sberbank.school.task02.util.FxRequest;
+import ru.sberbank.school.task02.util.FxResponse;
+import ru.sberbank.school.task02.exception.ConverterConfigurationException;
+
+import java.math.BigDecimal;
+import java.math.RoundingMode;
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Date;
+
+
+public class FxClientControllerImpl implements FxClientController {
+
+    private FxConversionService converter;
+
+    public static void main(@NonNull String[] args) {
+    //public static void main(String[] args) {
+        // FxRequest request = new FxRequest("usd/rub", "buy", "1000");
+        if (args.length != 3) {
+            throw new ConverterConfigurationException("Wrong arguments amount");
+        }
+        FxRequest request = new FxRequest(args[0], args[1], args[2]);
+        ServiceFactory factory = new ServiceFactoryImpl();
+        FxConversionService exCalc = factory.getFxConversionService(new ExternalQuotesServiceImpl());
+        FxClientController controller = new FxClientControllerImpl(exCalc);
+        System.out.println(controller.fetchResult(request));
+    }
+
+    public FxClientControllerImpl (FxConversionService converter) {
```

Comment:
&#39;(&#39; is preceded with whitespace.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a1803869115>*